content/blog/building-a-future-of-cloud-engineering/index.md

```diff
@@ -112,8 +112,8 @@ deployment statuses and to organize and find information more easily.
 It's never been such a breeze to go from idea to production.
 
 We've become enamored by this new approach to creating software that is
-truly born in the cloud. But it's not just our own belief -- it is
-working terrifically in practice. We have worked with organizations of
+truly born in the cloud. But it's not just our own belief -- this new age of cloud engineering is
+working terrifically in practice. We have worked with training and coaching in organizations of
```

Comment:
`We have worked with training and coaching` doesn't quite make sense.

Reply:
Those were topics that came up in our semantic analysis, but if the phrasing is too awkward it's fine to leave those out and shouldn't matter too much.